channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/reactions/TelephonyReactions.kt

```diff
@@ -1,25 +1,105 @@
 package com.justai.jaicf.channel.jaicp.reactions
 
+import com.justai.jaicf.channel.jaicp.dto.*
 import com.justai.jaicf.channel.jaicp.dto.AudioReply
 import com.justai.jaicf.channel.jaicp.dto.HangupReply
 import com.justai.jaicf.channel.jaicp.dto.SwitchReply
 import com.justai.jaicf.helpers.http.toUrl
 import com.justai.jaicf.logging.AudioReaction
 import com.justai.jaicf.reactions.Reactions
+import java.time.DayOfWeek
+import java.time.Instant
 
 val Reactions.telephony
     get() = this as? TelephonyReactions
 
 class TelephonyReactions : JaicpReactions() {
+    /**
+     * Appends audio to the response.
```

Comment:
I don't think we really need this javadoc here as it's the same as for base `Reactions` and doesn't describe some new telephony-related logic